### DISCUSSION TITLE
Welcome to Vahini-2.0 Discussions!

### DISCUSSION THREAD

**Discussion by Blake2912** [Blake2912/Vahini-2.0#5] [Announcements]
## 👋 Welcome!
Hi everyone, drop in all the resources that you find while you work on the project and let's keep it here. Let's have it in a single place where we can access it from anywhere

**Comment by Blake2912** [comment]
This would in turn help in easy handover and transition to juniors or whoever will work in the future

**Reply by amartyanambiar** [reply]
Yes
